about.html

```diff
@@ -0,0 +1,118 @@
+                <span class="navbar-toggler-icon"></span>
+              </button>
+              <div class="collapse navbar-collapse justify-content-end" id="navbarNavAltMarkup">
+                <div class="navbar-nav">
+                    <a class="nav-link active" aria-current="page" href="index.html">Home</a>
+                    <a class="nav-link" href="about.html">About Programme</a>
+                  <a class="nav-link" href="#">Designs</a>
+                  <a class="nav-link createborder" href="#">REGISTER NOW</a>
+                </div>
+              </div>
+            </div>
+          </nav>
+        </header>
+    </section>
+
+    <section class="aboutbg">
+        <div class="container">
+            <h1>GLOBBERY FASHION <br/> TRAINING PROGRAMME 2022</h1>
+            <p class="subtitle_abt">Our training programme will kick off with lots of exerpt and practical session,
+                you will learn the basis of cuting and sketching, Join the moving train and move your business to the 
+                next level. You can stop learning. If you do then fashion will move past you. 
+                We are also having different session with different sessoned professionals and designers all over the country.
+                Don't miss out on this opportunity to take your career to the next level.
+            </p>
+            <div class="bg-white-color">Please contact us more questions in Globbery fashion training 2022</div>
+            <div class="bg-white-color">services@globerry.org</div>
+        </div>
+    </section>
+
+
+    <section class="abt-session">
+        <div class="container">
+            <div class="row">
+                
+                <h6> Globbery Training and Fashion School 2022</h6>
+                <div class="col-12 exerpt-content  mx-auto">
+                   <p>This training is a once in a life time. Is a training that will make you a better person.
+                       Don't miss out on this opportunity of making it. Our programme will start with intense
+                       training session and continue to ensure that everything is going fine.
+                   </p>
+                 </div>
+
+                 <div class="col-12">
+                     <img src="./images/logo6.png" class="making-border img-fluid" alt="Globerry Logo"/>
+                 </div>
+                 
+            </div>     
+        </div>
+    </section>
+
+
+    <section class="past-trainings mx-auto">
+        <h3 class="text-center">See Past Globery Trainings</h3>
+        <p>Take a look at the two(2) Globerry Couture which took place at Lameridian Hotel, Igoja Lagos.</p>
+        
+            <img src="./images/20111.webp"  class="mx-auto img-fluid about-img" alt="Trainings"/>
+            <img src="./images/20112.jpg"  class=" mx-auto img-fluid about-img" alt="Trainings" />    
```

Comment:
- [ ] Hi. As it is recommended to follow the design, please kindly add a background color and a short text to your events(images)
**imp**
![image](https://user-images.githubusercontent.com/37901469/169526630-4c5cf22d-79ec-455b-9828-95e2f2587d82.png)
**design**
![image](https://user-images.githubusercontent.com/37901469/169526770-29fe0357-cbb0-4f3b-a09d-20428e4cf4fb.png)